python/ray/tune/examples/pbt_example.py

```diff
@@ -64,35 +90,29 @@ def reset_config(self, new_config):
         "--smoke-test", action="store_true", help="Finish quickly for testing")
     args, _ = parser.parse_known_args()
     if args.smoke_test:
-        ray.init(num_cpus=4)  # force pausing to happen for test
+        ray.init(num_cpus=2)  # force pausing to happen for test
     else:
         ray.init()
 
     pbt = PopulationBasedTraining(
         time_attr="training_iteration",
-        reward_attr="episode_reward_mean",
-        perturbation_interval=10,
+        reward_attr="mean_accuracy",
+        perturbation_interval=20,
         hyperparam_mutations={
-            # Allow for scaling-based perturbations, with a uniform backing
```

Comment:
Hm, there should be at least one example that showcases the many features of PBT. Maybe this triangle LR example should be a separate file?

Reply:
Hm I actually think it's better to have one good example. As a compromise, I added factor_2 back here. What do you think?

Reply:
I think this is fine for now; I think a better fix later is to have a very comprehensive walkthrough page in the Ray Examples section on how exactly to use PBT.